### PR TITLE
Lecture 15 (solr/ Sol Rodriguez)

### PR DESCRIPTION
**Pull request by SolRodriguez**

### CODE REVIEW

**Review by zhijian-liu — CHANGES_REQUESTED** [review]
Thanks for your contribution! However, your scribe notes are really hard to parse in its current form: https://github.com/SolRodriguez/6s965-fall2022/blob/main/notes/lecture-10-solr.md. Could you please refer to the scribe notes from other students and re-organize yours a bit? Feel free to re-request the review when it's ready. Thanks.

**Comment by SolRodriguez** [comment]
Lecture 15 Notes

**Comment by zhijian-liu** [comment]
@SolRodriguez , could you please separate the scribes for the two lectures into two separate PRs? Thank you!
